capplets/common/mate-theme-apply.c

```diff
@@ -48,12 +50,28 @@ void
 mate_meta_theme_set (MateThemeMetaInfo *meta_theme_info)
 {
   GSettings *interface_settings;
+  GSettings *interface_gnome_settings;
   GSettings *marco_settings;
   GSettings *mouse_settings;
   GSettings *notification_settings = NULL;
   gchar *old_key;
   gint old_key_int;
 
+  interface_gnome_settings = NULL;
```

Comment:
you could move this initialization at the declaration right above just like `notification_settings`.  Or keep it here, whichever you prefer.